cmd/segment-reaper/delete_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/private/testcontext"
+	"storj.io/storj/satellite/metainfo"
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/teststore"
+)
+
+func TestDeleteSegment(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	db := teststore.New()
+	defer ctx.Check(db.Close)
+
+	t.Run("segment is deleted", func(t *testing.T) {
+		err := makeSegment(ctx, db, "path1", time.Unix(10, 0))
+		require.NoError(t, err)
+
+		dryRun := false
+		deleteError := deleteSegment(ctx, db, "path1", time.Unix(10, 0), dryRun)
+		require.NoError(t, deleteError)
+		_, err = db.Get(ctx, storage.Key("path1"))
+		require.Error(t, err)
+		require.True(t, storage.ErrKeyNotFound.Has(err))
+	})
+	t.Run("segment is not deleted because of dryRun", func(t *testing.T) {
+		err := makeSegment(ctx, db, "path2", time.Unix(10, 0))
+		require.NoError(t, err)
+
+		dryRun := true
+		deleteError := deleteSegment(ctx, db, "path2", time.Unix(10, 0), dryRun)
+		require.NoError(t, deleteError)
+		_, err = db.Get(ctx, storage.Key("path2"))
+		require.NoError(t, err)
```

Comment:
Shouldn't we, just in case, assert that the returned value from this kind of `Get`s is the expected return value? Not only that there is no error.

Reply:
That we'd be good.
We can make the `makeSegment` helper function to return the expected value for after, being able to assert it.

Reply:
done